### ISSUE TITLE
After Upgrading to clearml-agent 3.4.0 getting error 

### ISSUE THREAD

**Issue by joebatish** [allegroai/clearml-helm-charts#167]
After upgrading to clearml-agent 3.4.0
when i am try o running tasks i am getting the following error :

Running kubectl encountered an error: error: error validating "/tmp/clearml_k8stmpl_wuzrqy8w.yml": error validating data: ValidationError(Pod.spec.hostAliases): invalid type for io.k8s.api.core.v1.PodSpec.hostAliases: got "map", expected "array"; if you choose to ignore these errors, turn validation off with --validate=false

```
`dependencies:
  - name: clearml
    version: 5.7.0
   
  - name: clearml-agent
    version: 3.4.0
    
`
```
this is not happened with 3.2.0

**Comment by valeriano-manassero** [comment]
Hi,
there was a bug in previous values.yaml that was fixed in 3.4.0. Are you sure you are not overriding this value somewhere?

pls check output of `helm get values --all <helm deployment name>` and let me know what is the value in `hostAliases` parameter.

**Comment by joebatish** [comment]
thanks @valeriano-manassero  i am closing this
